tuf/store/httpstore_test.go

```diff
@@ -303,27 +303,116 @@ func TestHTTPStoreRemoveAll(t *testing.T) {
 	server := httptest.NewServer(http.HandlerFunc(handler))
 	defer server.Close()
 	store, err := NewHTTPStore(server.URL, "metadata", "json", "targets", "key", http.DefaultTransport)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 
 	// currently unsupported since there is no use case
 	// check for the error
 	err = store.RemoveAll()
-	assert.Error(t, err)
+	require.Error(t, err)
 }
 
 func TestHTTPOffline(t *testing.T) {
 	s, err := NewHTTPStore("https://localhost/", "", "", "", "", nil)
-	assert.NoError(t, err)
-	assert.IsType(t, &OfflineStore{}, s)
+	require.NoError(t, err)
+	require.IsType(t, &OfflineStore{}, s)
 }
 
 func TestErrServerUnavailable(t *testing.T) {
 	for i := 200; i < 600; i++ {
 		err := ErrServerUnavailable{code: i}
 		if i == 401 {
-			assert.Contains(t, err.Error(), "not authorized")
+			require.Contains(t, err.Error(), "not authorized")
 		} else {
-			assert.Contains(t, err.Error(), "unable to reach trust server")
+			require.Contains(t, err.Error(), "unable to reach trust server")
 		}
 	}
 }
+
+// If successful, GetKey and RotateKey both return public keys
```

Comment:
This comment was slightly confusing when I first read it, maybe could we reword it to:
`GetKey and RotateKey both succeed if valid public keys are returned from the server`?